src/server/utilities.ts

```diff
@@ -173,10 +173,15 @@ namespace ts.server {
     export function createSortedArray<T>(): SortedArray<T> {
         return [] as SortedArray<T>;
     }
+}
 
+/* @internal */
+namespace ts.server {
     export class ThrottledOperations {
-        private pendingTimeouts: Map<any> = createMap<any>();
-        constructor(private readonly host: ServerHost) {
+        private readonly pendingTimeouts: Map<any> = createMap<any>();

```

Comment:
Use `Map<{}>` to avoid unsafety

Reply:
why? how is that better? the set time out anyways return any and using {} instead of any isnt going to help?

Reply:
`any` implies something that we haven't yet typed, but which we are assuming to have some particular type. `{}` indicates that we aren't depending on it having a particular type anywhere.
```ts
function f(x: any): number {
    return x.prop; // Compiler says: OK, whatever, hope you know what you're doing
}

function g(x: {}): number {
    return x.prop; // Compiler says: But you said you didn't care what `x` is, now you expect `prop: number`!
}
```